### PR TITLE
keysController field names

### PR DESCRIPTION
**Pull request by jmank88**

### CODE REVIEW

**Review by obelisk — APPROVED** [review]

File: core/web/keys_controller.go [hunk 1]
```diff
@@ -1,6 +1,7 @@
 package web
 
 import (
+	"fmt"
```
Comment (obelisk) [inline core/web/keys_controller.go:4]: is there a reason we can't use `errors.Errorf`?

Comment (obelisk) [reply core/web/keys_controller.go:4]: If so, is there any benefit to doing so?

Comment (jmank88) [reply core/web/keys_controller.go:4]: :shrug: Would it be an improvement in this case? The panic will already have the same stack trace.

Comment (jmank88) [reply core/web/keys_controller.go:4]: panic doesn't even need an error actually - it's quite common to pass just a formatted string.

**Pull request merged**